cnames_active.js

```diff
@@ -1740,6 +1740,7 @@ var cnames_active = {
   "max": "toorusr.github.io/max.js.org",
   "maxnachlinger": "maxnachlinger.github.io", // noCF? (don´t add this in a new PR)
   "maxtracking": "maxtracking.github.io", // noCF? (don´t add this in a new PR)
+  "maxvalid":"techmonowar.github.io/maxvalid",
```

Comment:
```suggestion
  "maxvalid": "techmonowar.github.io/maxvalid",
```
As the CI error notes, you're missing a space here